content/blog/using-mcps-to-run-terraform.md

```
@@ -0,0 +1,206 @@
+  owner    = postgresql_role.app_role.name  
+  database = postgresql_database.app_db.name  
+}
+```
+
+<br>
+
+### Debugging terraform apply
+At this point, I was ready for the agent to run terraform plan and apply my changes. And here is where the rubber meets the road in using MCPs 🙂. Here’s what I ran into:
+
+**The first issue** was a permissions error — `admin_user` couldn’t create Postgres roles. This issue was totally my fault — I hadn’t granted it any meaningful privileges. I had the MCP repeatedly run `terraform apply`, but either the MCP didn't log/surface the error or Cursor’s Agent Mode wasn't able to interepret the cause of the error.
+![First issue - MCP able to run terraform apply][/img/updates/using-mcps-to-run-terraform/debug1.png]
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "/img/updates/using-mcps-to-run-terraform/debug1.png"

(MD052, reference-links-images)